site.yml

```diff
@@ -60,3 +60,30 @@
   - { role: swarm, run_as: worker }
   - { role: contiv_network, run_as: worker }
   - { role: contiv_storage, run_as: worker }
+
+# net-master hosts correspond to cluster machines that run the master
+# logic of the contiv net services
+- hosts: net-master
+  sudo: true
+  environment: env
+  roles:
+  - { role: base }
```

Comment:
even for provisioning contiv services on a baremetal, we should be using a baremetal that is setup using `contiv/lab` and that I think falls in purview of this PR. 

I think we need to agree on a production environment usecase. And that usecase in my mind is that a user brings up a machine with a base os image (which is already provisioned using the `base` role) and then provisions contiv services on top. Do we agree?